languages/rust/exercises/concept/tuples/.docs/hints.md

```diff
@@ -0,0 +1,21 @@
+## General
+
+- <https://doc.rust-lang.org/book/ch03-02-data-types.html#the-tuple-type>
+- <https://doc.rust-lang.org/stable/rust-by-example/primitives/tuples.html>
+- <https://doc.rust-lang.org/stable/rust-by-example/flow_control/match/destructuring.html>
```

Comment:
Maybe considering adding labels to these links? This will make a bit friendlier to students when being displayed on the website.

Reply:
bbafaaf